backend/db/index.js

```diff
@@ -25,6 +25,9 @@ module.exports = {
       if (collections[collection].timeseries) {
         // If the collection should be a time series, add the current date
         const now = new Date()
+        doc.ts = new Date(Date.UTC(
```

Comment:
I think you could use `Date.now()` here :)

Reply:
That seems weird if the date value is UTC

Reply:
Let's keep this for now to be sure